core/organisations.go

```diff
@@ -51,6 +51,8 @@ type OrganisationService interface {
 	ListOrganisationsForUser(ctx context.Context, id UserID) ([]Organisation, error)
 	// List the users that belong to an organisation or ErrOrganisationDoesNotExist if no such organisation exists
 	ListUsersInOrganisation(ctx context.Context, id OrganisationID) ([]User, error)
+	// Check if a member exists for the given email
+	IsMemberByEmail(ctx context.Context, OrgID OrganisationID, email EmailAddress) bool
```

Comment:
Kan je daar "HasMemberWithEmail" van maken? dat lijkt me iets duidelijker. Je wilt ook altijd een error teruggeven in services omdat die heel vaak over externe connecties gaan lopen. Het is aan de stories om te bepalen of ze die error willen doorgeven of dat als "false" of "true" willen beschouwen voor hun interne logica

Reply:
Ik denk dat het misschien ook logischer is als dit een Get wordt en die de user zelf teruggeeft? Stories kunnen er dan nog altijd zelf een boolean van maken als ze dat willen. In tegenstelling tot Wachtplanner willen we in Apollo services wel meer veralgemenen aangezien die in meerdere applicaties gebruikt moeten kunnen worden 😛